src/main/cpp/src/parse_uri.cu

```diff
@@ -951,36 +954,85 @@ std::unique_ptr<column> parse_uri(strings_column_view const& input,
                              std::move(null_mask));
 }
 
+void validate_output(strings_column_view const& input,
+                     column const& result,
```

Comment:
Output is unused, so this validation is wrong.

Reply:
In addition, never use `column` directly. Use `column_view` instead.

Reply:
Thanks for catching this. The function name is confusing. This function would validate the input urls (non nulls). If the input url  is invalid then the validity vector in this function is set. We actually don't compare the output in this case. I will change the function name and the parameters to this function.